src/ui/components/tiles/HorizontalOfferTile.native.test.tsx

```diff
@@ -151,21 +156,46 @@ describe('HorizontalOfferTile component', () => {
   })
 
   describe('When offer is a `SEANCE_CINE`', () => {
+    const OCTOBER_5_2020 = 1601856000
+    const NOVEMBER_1_2020 = new Date(2020, 10, 1) // This date is used as now
+    const NOVEMBER_12_2020 = 1605139200
```

Comment:
je trouve qu'un timestamp est peu lisible (heureusement, il y a la variable bien nommée)

est-ce qu'il y aurait des contre-indications à utiliser `Date` ?
```suggestion
    const OCTOBER_5_2020 = new Date(2020, 9, 5).getTime() / 1000
    const NOVEMBER_1_2020 = new Date(2020, 10, 1) // This date is used as now
    const NOVEMBER_12_2020 = new Date(2020, 10, 12).getTime() / 1000
```

Reply:
C'est ce que j'avais mis au début 😂 mais j'ai trouvé que vu que les variables été clairement nommées, je pouvais laisser